glean-core/src/metrics/string_list.rs

```diff
@@ -133,4 +134,17 @@ impl StringListMetric {
         let value = Metric::StringList(value);
         glean.storage().record(&self.meta, &value);
     }
+
+    /// **Test-only API (exported for FFI purposes).**
+    ///
+    /// Get the currently-stored values as a JSON String.
+    ///
+    /// This doesn't clear the stored value.
+    pub fn test_get_value_as_string(&self, glean: &Glean, storage_name: &str) -> Option<String> {
```

Comment:
Follow up from below: Let's implement a `test_get_value` function returning `Vec<String>` to use in Rust tests.

this `as_string` function can then call that as well.